src/views/download/download.jsx

```diff
@@ -0,0 +1,171 @@
+var React = require('react');
+var render = require('../../../lib/render.jsx');
+
+var FormattedHTMLMessage = require('react-intl').FormattedHTMLMessage;
+var FormattedMessage = require('react-intl').FormattedMessage;
+
+var Page = require('../../../components/page/www/page.jsx');
+var TitleBanner = require('../../components/title-banner/title-banner.jsx');
+var FlexRow = require('../../components/flex-row/flex-row.jsx');
+var SubNavigation = require('../../components/subnavigation/subnavigation.jsx');
+
+require('./download.scss');
+
+var Download = React.createClass({
+    type: 'Download',
+    render: function () {
+        return (
+            <div className="download">
+                <TitleBanner className="masthead">
+                    <div className="inner">
+                        <h1 className="title-banner-h1">
+                            <FormattedMessage id='download.title' />
+                        </h1>
+                        <p className="title-banner-p intro">
+                            <FormattedMessage id='download.intro' />
+                        </p>
+                        <p className="title-banner-p intro">
+                            <FormattedHTMLMessage id='download.introMac' />
+                        </p>
+                    </div>
+                    <div className="band">
+                        <SubNavigation className="inner">
+                            <a href="#installation">
+                                <li>
+                                    <FormattedMessage id='download.installation' />
+                                </li>
+                            </a>
+                            <a href="#updates">
+                                <li>
+                                    <FormattedMessage id='download.updatesTitle' />
+                                </li>
+                            </a>
+                            <a href="#other">
+                                <li>
+                                    <FormattedMessage id='download.otherVersionsTitle' />
+                                </li>
+                            </a>
+                            <a href="#issues">
+                                <li>
+                                    <FormattedMessage id='download.knownIssuesTitle' />
+                                </li>
+                            </a>
+                        </SubNavigation>
+                    </div>
+                </TitleBanner>
+                <div className="inner">
+                    <section id="installation">
+                        <span className="nav-spacer"></span>
+                        <FlexRow className="three-col-row">
```

Comment:
I would probably do max-width of 4 columns (i.e. `18.75rem`), with a bit of padding to ensure the text doesn't run into each other. Then maybe change the `jusify-content` on the container to be `space-between` so that it fills the whole space.

As for on tablet and mobile – hmm. Not sure. Maybe make it `12.5rem` for tablet, and then back to `18.75rem` for mobile in landscape and portrait